ci/docker/connector_build/Dockerfile

```
@@ -1,9 +1,11 @@
 # We use manylinux1 base image because pyarrow_manylinux2010 has a bug and wheel failed to be audited
-ARG BASE_IMAGE=quay.io/pypa/manylinux2010_x86_64
+ARG BASE_IMAGE
 FROM $BASE_IMAGE
 
 # This is to solve permission issue, read https://denibertovic.com/posts/handling-permissions-with-docker-volumes/
-RUN curl -o /usr/local/bin/gosu -SL "https://github.com/tianon/gosu/releases/download/1.11/gosu-amd64"
+ARG GOSU_URL
```

Default to old values
```suggestion
ARG GOSU_URL=https://github.com/tianon/gosu/releases/download/1.11/gosu-amd64
```